app/_hub/nonamesecurity/nonamesecurity-kongtrafficsource/overview/_index.md

```diff
@@ -16,6 +16,11 @@ For example, an action could be fetching a log file, or to initiating a block ba
 The Noname Security Kong plugin is available as a LuaRocks module.
 The [Noname Security install and configuration documentation](https://docs.nonamesecurity.com/docs/kong-plugin) explains how to log on to the Noname admin user interface, go to the Integrations section of the platform, begin the Kong integration wizard, and download the plugin that is used in the Dockerfile to create a custom docker image with the plugins preinstalled. 
 
+Prevention is enabled by default. To disable the prevention feature, see [official documentation](https://docs.nonamesecurity.com/docs/kong-plugin#disabling-the-prevention-feature) 
```

Comment:
```suggestion
Prevention is enabled by default. To disable the prevention feature, review the [official documentation](https://docs.nonamesecurity.com/docs/kong-plugin#disabling-the-prevention-feature).
```
@Guaris added this review comment earlier but looks like it got lost along the way, so adding it again.